Build/PreBuild.ps1

```diff
@@ -8,6 +8,7 @@ $need_symlink = @(
     "python2",
     "MicroBuild.Core",
     "Microsoft.VSSDK.BuildTools",
+    "Microsoft.VSSDK.Debugger.VSDConfigTool",
```

Comment:
Why?

Reply:
vsdconfig executable was pulled out of the build tools package and moved into its own package. See the email I sent yesterday.